src/utilities/network/lookupApi.js

```diff
@@ -22,6 +22,34 @@ const SEPARATORS = {
   POLICY_NUMBER: '-',
 };
 
+export const createInsurancePolicyRecord = response => {
+  const {
+    id,
+    policyNumberFamily,
+    policyNumberPerson,
+    type,
+    discountRate,
+    expiryDate,
+    enteredById,
+    nameId,
+    insuranceProviderId,
+    isActive,
+  } = response;
+  const policy = { 
+    id,
+    policyNumberFamily,
+    policyNumberPerson,
+    type,
+    discountRate: parseNumber(discountRate),
+    expiryDate: parseDate(expiryDate),
+    enteredBy: UIDatabase.getOrCreate('User', enteredById),
+    patient: UIDatabase.getOrCreate('Name', nameId),
+    insuranceProvider: UIDatabase.getOrCreate('InsuranceProvider', insuranceProviderId),
+    isActive: parseBoolean(isActive)
+  };
+  UIDatabase.write(() => UIDatabase.update('InsurancePolicy', policy));
+};
```

Comment:
also if we're integrating insurance policies from other stores we need a mechanism to ensure they're read only. An easy solution would be to add a getter to `InsurancePolicy`?

```
get canEdit(){
    return this.patient.thisStoresPatient
}
```

and then obviously the code on the policy edit to ensure it's read only?

Is there any other patient data being integrated into mobile?

Reply:
and possibly to be extra safe adding a check in outgoing sync to ensure policies which aren't from this stores patients are never synced out

Reply:
> also if we're integrating insurance policies from other stores we need a mechanism to ensure they're read only

#2543 :).
 
> and possibly to be extra safe adding a check in outgoing sync to ensure policies which aren't from this stores patients are never synced out

#2544 :).